myokit/_sim/cmodel.py

```diff
@@ -137,6 +139,45 @@ def __init__(self, model, sensitivities):
         self.parameters = parameters
         self.parameter_derived = parameter_derived
 
+    def _prepare_bindings(self, model, labels):
```

Comment:
I was imagining this either getting inlined (and possibly simplified?) anywhere we used it, or becoming a private method somewhere. Happy with this for now, but definitely part of the bigger project to get rid of or hide this somehow